nix/pkgs/status-go/library/default.nix

```diff
@@ -58,9 +81,14 @@ in pkgs.buildGoModule {
   # Also set CLEANUP_GENERATED_FILES_DRY_RUN=true to avoid running cleanup_generated_files.sh script,
   # which is not available at this phase, because buildGoModule only copies Go files.
   buildPhase = ''
+    runHook preBuild
```

Comment:
hmm, I was fighting with this part, because explicit  `runHook preBuild` here means that `make generate` is invoked one more time, which is redundant.

You probably had to bring it back because you need `export LIBS_DIR="${codexLib}"` here. I think it's better to just copy `LIBS_DIR` definition here.